app/controllers/scoreboards_controller.rb

```diff
@@ -5,13 +5,17 @@ class ScoreboardsController < ApplicationController
 
   # GET /scoreboards
   def index
-    @scoreboard_all = ScoreboardAll.new
+    @national_scoreboard = NationalScoreboard.new
     @scoreboards = ComparisonService.new(current_user).list_scoreboards
   end
 
   def show
-    if params[:id] == ScoreboardAll::SLUG
-      @scoreboard = ScoreboardAll.new
+    case params[:id]
+    when 'all'
```

Comment:
minor thing but the redirect could be done in the routes instead?

Reply:
guess the advantage is keeping logic out of the controller but it also seems convenient to see it together here too